src/app/shared/form/builder/parsers/field-parser.ts

```diff
@@ -310,7 +310,7 @@ export abstract class FieldParser {
     }
   }
 
-  protected initModel(id?: string, label = true, labelEmpty = false, setErrors = true, hint = true) {
+  protected initModel(id?: string, label = true, labelEmpty = false, setErrors = true, hint = true, help = true) {
```

Comment:
maybe use a more explicit variable name like 'showHelp' instead of just 'help'

Reply:
`help` is the name the field (like 'hint', I keep the same logic)